source/Core/Src/gui.cpp

```diff
@@ -267,10 +265,8 @@ static void printShortDescription(uint32_t shortDescIndex, uint16_t cursorCharPo
   printShortDescriptionDoubleLine(shortDescIndex);
 
   // prepare cursor for value
-  OLED::setFont(0);
-  OLED::setCharCursor(cursorCharPosition, 0);
   // make room for scroll indicator
-  OLED::setCursor(OLED::getCursorX() - 2, 0);
+  OLED::setCursor(cursorCharPosition * FONT_12_WIDTH - 2, 0);
```

Comment:
Do you think its worth us expanding setCursor to take the font type as well (and have a value for pixels instead of a font position)? Thinking only for aesthetics really. Though could be nice to keep the font dimensions contained inside `OLED::`?

Reply:
>However, at this point I think just hardcoding the cursor position is fine... I mean there is always room for more refactoring in the entire codebase.

Point taken; I agree there is always more refactoring to be done. Was just curious so hear your thoughts on this incase it was trivial or not.

I think release this as is for now and if we need to we can refactor later on.